src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,19 +64,69 @@ public RobotContainer() {
      * passing it to a {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
-        swerveSubsystem.setDefaultCommand(new RunCommand(() -> {
-            double xPower = -driveController.getLeftY();
-            double yPower = -driveController.getLeftX();
-            double angularPower = -driveController.getRightX();
-            swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower);
-        }, swerveSubsystem));
     }
 
     /**
      * Use this to pass the autonomous command to the main {@link Robot} class.
+     * 
      * @return the command to run in autonomous
      */
     public Command getAutonomousCommand() {
         return autonChooser.getSelected();
     }
+
+    public void periodic() {
```

Comment:
Default commands :( they're easier to read! Plus less confusing since you technically want this method to be a `teleopPeriodic()`, not just a regular `periodic()` loop that also runs during auton. Plus using button callbacks handles debounce and other oddities; doing it manually just seems like theres a lot more room for problems to arise.

Reply:
Ex.
```java
// Periodic loop
if (mech.getRightBumperPressed()) {
   elevator.height = elevator.height.next();
}
if (mech.getLeftBumperPressed()) {
    elevator.height = elevator.height.previous();
}
```
```
// Button callbacks in `configureButtonBindings()`
// Made even cleaner with methods and method references (ie. `elevator::nextPosition` or `elevator::previousPosition`)
mechRightBumper.whenPressed(new InstantCommand(() -> elevator.height = elevator.height.next()));
mechLeftBumper.whenPressed(new InstantCommand(() -> elevator.height = elevator.height.previous()));
```